### DISCUSSION TITLE
Question about setting for KENs

### DISCUSSION THREAD

**Discussion by overflow218** [ANLAB-KAIST/KENSv3#22] [Project Q&A 1. Application Layer (Socket)] [answered]
조교님 안녕하세요 KENS 초기 설정과 관련하여 질문드립니다.
저는 윈도우를 사용중이고, 다른 수업에서 받은 VM을 이용하여 과제를 진행하려고 합니다.
적어주신 글을 읽고 어찌어찌해서 make 빌드까지 성공했습니다.
다만 $ cmake -B build . 를 하면 다음과 같은 에러가 발생하는데 괜찮은가요?
VM 우분투 버전이 18.04라서 다른 패키지들도 약간 버전이 낮아서 나오는 경고같긴한데 혹시 진행에 문제가 있진 않을까 여쭤봅니다.
혹시 몰라 $ ./app/echo/echo 를 실행해봤는데 잘 작동하는 것 같습니다.

Hello TA!
I'm wondering about setting for KENS.
I'm using Windows, so I'm trying to do assignments with VM that I got from other class.
I manages to success make build on KENs.
However, when I type $ cmake -B build . , It gives me warning like picture.
I think It's because I'm using ubuntu version 18.04 and other lower version packages.
I tried $ ./app/echo/echo and It looks working.
will be there any problems about it?
Thank you!
![image](https://user-images.githubusercontent.com/48017554/158381036-dd882c42-1f4d-4411-8975-baf613223202.png)

**Comment by Pusnow** [comment, marked as answer]
Both warning messages are from GoogleTest library, and they are okay to dismiss.
